napari/_vispy/vispy_surface_layer.py

```diff
@@ -63,13 +57,13 @@ def _on_colormap_change(self, event=None):
             )
         self.node.cmap = cmap
 
-    def _on_contrast_limits_change(self, event=None):
-        self.node.clim = self.layer.contrast_limits
+    def _on_contrast_limits_change(self, contrast_limits):
```

Comment:
Can you add a doc string - can be identical to this https://github.com/napari/napari/blob/master/napari/_vispy/vispy_image_layer.py#L165-L171